docs/products/tools/marketplace/guides/passbolt/index.md

```diff
@@ -0,0 +1,86 @@
+---
+slug: Deploy Passbolt Through The Linode Marketplace
+title: "Deploy Passbolt Through The Linode Marketplace"
+description: 'Deploy Passbolt password manager through the Linode Marketplace.'
+og_description: 'Deploy Passbolt password manager through the Linode Marketplace.'
+keywords: ['passbolt','password manager','security','authentication']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+authors: ["Linode"]
+published: 2024-02-05
+modified_by:
+  name: Linode
+---
+
+[Passbolt Password Manager](https://github.com/passbolt/passbolt_api) is an open-source password manager designed for teams and businesses. It allows users to securely store, share and manage passwords.
+
+## Deploying a Marketplace App
+
+{{< content "deploy-marketplace-apps-shortguide">}}
+
+{{< content "marketplace-verify-standard-shortguide">}}
+
+{{< note >}}
+**Estimated deployment time:** Passbolt should be fully installed within 5-10 minutes after the Compute Instance has finished provisioning.
+{{< /note >}}
+
+## Configuration Options
+
+- **Supported distributions:** Ubuntu 22.04 LTS
+- **Recommended plan:** We recommend a 4GB Dedicated CPU or Shared Compute instance for Passbolt.
+
+### Passbolt Options
+
+{{< content "marketplace-required-limited-user-fields-shortguide">}}
+
+## Getting Started after Deployment
+
+To start registration follow the link provided in `/etc/motd` to visit the DNS you choose during deployment. 
+```
+cat /etc/motd
+*********************************************************
+Akamai Connected Cloud passbolt Marketplace App
+
+Registering admin user: https://$DNS_NAME/setup/start/$UUID
+
+App URL: https://$DNS_NAME
+Credentials File: /home/$SUDO_USER/.credentials
+Documentation: https://www.linode.com/marketplace/apps/linode/passbolt/
+*********************************************************
+To delete this message of the day: rm /etc/motd
+```
+
+
+Be sure to download the `passbolt-recovery-kit.txt` file and store it in a safe place. This PGP Private Key Block  (and the passphrase you set here) will be required if you ever need to go through account recovery.
+
+You will need to create a passphrase to access Passbolt 
+
+    ![Screenshot of Passbolt passphrase](passphrase_screen.jpg)
+
+Once the passphrase is set you will be able to view the Passbolt main screen. 
+
+    ![Screenshot of Passbolt main screen](passbolt_mainscreen.jpg)
+
+Please visit [Passbolt Installation Documentation](https://help.passbolt.com/hosting/install/ce/ubuntu/ubuntu.html) for information on how to configure Passbolt.
```

Comment:
Suggest:

"For information on how to configure Passbolt, visit [Passbolt Installation Documentation](https://help.passbolt.com/hosting/install/ce/ubuntu/ubuntu.html)."